solutions/swb-reference/end-to-end-tests/config/testEnv.yaml

```diff
@@ -0,0 +1,6 @@
+AdminUser: 'aws-chamonix-dev+testenvroot@amazon.com'
```

Comment:
are you sure you want this email checked in?

Reply:
I was having double thoughts because is an email but in the end is a user for e2e testing, so I will move it to secrets

Reply:
moved to secrets